wagtail/contrib/simple_translation/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db.models import Model
+
+
+class SimpleTranslation(Model):
```

Comment:
Wonder if it's worth having this permission in core so that it could be shared? But leave it up to the translation packages to actually register it so it won't appear in the admin unless you have one installed?

Reply:
I added the permission to https://github.com/wagtail/wagtail/blob/main/wagtail/admin/models.py#L21

I like the idea because you do not have to migrate anymore. The permission will exist before adding simple_translation to installed apps.

Is this what you where looking for?

Reply:
Yep, thanks!